sdk-core/build.gradle

```diff
@@ -2,51 +2,15 @@ apply plugin: "com.android.library"
 apply plugin: "kotlin-android"
 apply plugin: "kotlin-android-extensions"
 
-group = lib_group
-version = lib_version
-
-android {
-    compileSdkVersion compile_sdk_version
-
-    defaultConfig {
-        minSdkVersion min_sdk_version
-        targetSdkVersion target_sdk_version
-        versionCode version_code
-        versionName this.version
-    }
-
-    buildTypes {
-        release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro"
-            consumerProguardFiles 'proguard-rules.pro'
-        }
-    }
-
-    kotlinOptions {
-        freeCompilerArgs = ["-Xallow-result-return-type"]
-    }
-
-    androidExtensions {
-        /**
-         *  Used to create automatically the @Parcelize methods
-         *  Link: https://kotlinlang.org/docs/tutorials/android-plugin.html#parcelable-implementations-generator
-         */
-        experimental = true
-    }
-}
+def ext = rootProject.ext
+def libs = ext.libraries
 
 dependencies {
-    implementation "androidx.appcompat:appcompat:1.1.0"
-
-    implementation "com.android.support:customtabs:28.0.0"
-    implementation "com.github.kittinunf.result:result:2.2.0"
-    implementation "com.google.android.gms:play-services-fido:18.1.0"
+    implementation ext.kotlin_stdlib
+    implementation libs.androidCustomTabs
+    implementation libs.result
+    implementation libs.playServicesFido
+    implementation libs.retrofit
+    implementation libs.retrofitConverterGson
     implementation "com.squareup.okhttp3:logging-interceptor:4.0.0"
```

Comment:
Can this also be moved to `ext.libraries` like the others ?

Reply:
This library is only used within this module, so I preferred to not include it there.